src/client/components/WorkHistoryTile.js

```diff
@@ -12,29 +12,19 @@ const WorkHistoryTile = ({ workHistoryData, openModal, removeItem }) => (
       <Text.Block
         size="large"
         css={`
-          font-weight: 700;
+          line-height: 1.8;
+          font-weight: bold;
```

Comment:
Same question here: do we need this?

And it looks like we're overriding the styles of `Text.Block` in the same way in two places. If we do need these overrides, is it time to extract a new component?

Reply:
I draw my line at 3 occurrences ;) But good point, I added new component for them.